meinberlin/apps/plans/views.py

```diff
@@ -104,7 +105,7 @@ def get_context_data(self, **kwargs):
 
         if (hasattr(settings, 'A4_USE_VECTORMAP') and
                 settings.A4_USE_VECTORMAP):
-            use_vector_map = 1
+            use_vector_map = 12
```

Comment:
I am not sure, I was already allowed to look. But anyway: this shouldn't be changed. :)

Reply:
That was a mistake - I kept changing the pagination to check if it worked from 12 to 1. Sorry about this